### PR TITLE
Good resources to learn capture the flags competitions - Cybersecurity+Linux

### PR DESCRIPTION
**Pull request by namanjain007**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Add info

### Description 
 CTFs are events that are usually hosted at information security conferences, including the various BSides events. These events consist of a series of challenges that vary in their degree of difficulty, and that require participants to exercise different skillsets to solve. Once an individual challenge is solved, a “flag” is given to the player and they submit this flag to the CTF server to earn points. Players can be lone wolves who attempt the various challenges by themselves, or they can work with others to attempt to score the highest number of points as a team.

### Why is this valuable (or not)?
 CTF is a great hobby for those interested in problem-solving and/or cyber security. The community is always welcoming and it can be a lot of fun tackling challenges with friends

### How do we know it's really free?
All the resources are of CMU picoctf , toutube free channels and some free web hosting sites.

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by eshellman** [comment]
I'm not sure "courses" is the right place for this - some of the resources are not courses and might fit on the books page. The competitive programming page might be a better choice.

**Comment by namanjain007** [comment]
Please merge the pull now.

File: free-programming-books.md [hunk 1]
```diff
@@ -1169,6 +1170,16 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Crystal for Rubyists](http://www.crystalforrubyists.com)
 
 
+### ctf
+
+* [Binary Exploitation picoctf by Carnegie Mellon University](https://picoctf.org/learning_guides/Book-5-Binary-Exploitation.pdf) (PDF)
+* [Capture the flags introduction and video tutorials to various events](https://www.youtube.com/channel/UClcE-kVhqyiHCcjYwcpfj9w) (PDF)
```
Comment (eshellman) [inline free-programming-books.md:1176]: not a book

**Comment by namanjain007** [comment]
removed the non-book elemenr.

**Comment by namanjain007** [comment]
review this pull...

File: problem-sets-competitive-programming.md [hunk 2]
```diff
@@ -34,6 +35,12 @@
 * [URI Online Judge](https://www.urionlinejudge.com.br/judge/en/login)
 
 
+### ctf
```
Comment (eshellman) [inline problem-sets-competitive-programming.md:38]: use the full heading (need to change the link to correspond

**Review by eshellman — CHANGES_REQUESTED** [review]
This is a valuable addition, even if it doesn't fit neatly. Thank you for sticking with it; we've had a large volume of submissions.

File: problem-sets-competitive-programming.md (hunk 2 above)
Comment (namanjain007) [reply problem-sets-competitive-programming.md:38]: okay , makes sense.

Comment (eshellman) [reply problem-sets-competitive-programming.md]: whT bout the ctf links in security?

Comment (namanjain007) [reply problem-sets-competitive-programming.md]: those links were more of practice and pentesting labs arena which i added in competitive programming section.

Comment (namanjain007) [reply problem-sets-competitive-programming.md]: I will add more resources in future as I myself have started doing ctf few months back.

Comment (namanjain007) [reply problem-sets-competitive-programming.md]: Great initiative !!!

**Comment by eshellman** [comment]
I think it's time to merge this; I'm sill wondering about the two security items that seem to be ctf items, but that can be another PR. Thanks, and welcome to free-programming-books!

**Pull request merged**